.github/workflows/build_rpms.yml

```diff
@@ -0,0 +1,39 @@
+name: RPM Build
+
+on: [pull_request]
```

Comment:
Add `push` here as well? github-actions storage is free for us, I think.

Reply:
Sure, I can do that, @dvdhrm.